pages/complete-signup.tsx

```diff
@@ -3,8 +3,18 @@ import CompleteSignup from '../src/features/user/CompleteSignup';
 import tenantConfig from '../tenant.config';
 import Head from 'next/head';
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import { useTranslation } from 'next-i18next';
 
 export default function UserProfile() {
+  const { i18n } = useTranslation();
+
+  React.useEffect(() => {
+    if (localStorage.getItem('i18nextLng') !== null && i18n) {
+      const languageFromLocalStorage: any = localStorage.getItem('i18nextLng');
+      i18n.changeLanguage(languageFromLocalStorage);
+    }
+  }, [i18n]);
```

Comment:
There are still some pages left which are shown in English where you have not added this, e.g. `api-key.tsx`, `delete-account.tsx` et al. Please check all pages! I wonder if it is really necessary to implement this for every page in the pages directory or if it can be solved this with some kind of abstraction?

Reply:
I vote for trying to solve it generally instead of adding the code to every single page. Check out https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1596#discussion_r1014614191 below.

Reply:
I checked each page and now translation is working properly for all